ant-protocol/src/storage/graph.rs

```diff
@@ -22,7 +22,7 @@ pub struct GraphEntry {
     pub owner: PublicKey,
     pub parents: Vec<PublicKey>,
     pub content: GraphContent,
-    pub outputs: Option<Vec<(PublicKey, GraphContent)>>,
+    pub outputs: Vec<(PublicKey, GraphContent)>,
     /// signs the above 4 fields with the owners key
```

Comment:
The reason for option here is that some patterns we do not have outputs, only Parent. i.e. when used as a file change history the GraphEntry can only go backwards. So the content is a direct or whatever and changes with each entry. There are no outputs in that pattern though.

Reply:
An empty vec can be considered as a pattern. But an `Option` makes it more obvious, though might be a little more complex handling in code.

Reply:
Empty vec indeed sounds like the right balance between usability and simplicity, what do you think?